src/commands/util/event.js

```diff
@@ -0,0 +1,56 @@
+const Command = require('../Command');
+const fetch = require('node-fetch');
+
+const info = ':information_source:';
+const calendar = ':calendar_spiral:';
+const pushPin = ':round_pushpin';
+const clock = ':clock3:';
+
+module.exports = new Command({
+  name: 'event',
+  description: 'get number of events from a calendar',
+  category: 'information',
+  aliases: [],
+  permissions: 'general',
+
+  execute: async (message, args) => {
```

Comment:
because we are not gonna use `await` anymore
```suggestion
  execute: (message, args) => {
```